docs/en/query_language/agg_functions/reference.md

```diff
@@ -1,11 +1,65 @@
-# Function reference
+# Function Reference
 
-## count() {#agg_function-count}
+## count {#agg_function-count}
 
-Counts the number of rows. Accepts zero arguments and returns UInt64.
-The syntax `COUNT(DISTINCT x)` is not supported. The separate `uniq` aggregate function exists for this purpose.
+Counts the number of rows.
+
+```
+count(expr)
+```
+
+**Parameters**
+
+The function can take zero or one parameter.
+
+- If the function is called without parameters it counts the number of rows. The `count()` syntax is a ClickHouse-specific implementation. The most other DBMS suggest `COUNT(*)` usage.
```

Comment:
`*` and no arguments should look like equal, from current version it's not even clear that ClickHouse supports standard syntax as well